deploy/helm/sumologic/values.yaml

```diff
@@ -2381,6 +2469,173 @@ otelagent:
 
 ## Configure otelcol
 otelcol:
+  securityContext:
+    ## ToDo: Verify following comment
+    ## The group ID of all processes in the statefulset containers. By default this needs to be otelcol(999).
+    fsGroup: 999
+
+  ## Add custom labels to all otelcol sts pods(logs, metrics, events)
+  podLabels: {}
+
+  ## Add custom annotations to all otelcol sts pods(logs, metrics, events)
+  podAnnotations: {}
+
+  ## Add custom labels to all otelcol svc (logs, metrics, events)
+  serviceLabels: {}
+
+  ## Add custom labels to all otelcol statefulset PVC (logs, metrics, events)
+  pvcLabels: {}
+
+  statefulset:
+    image:
+      repository: public.ecr.aws/sumologic/sumologic-otel-collector
+      tag: 0.0.18
+      pullPolicy: IfNotPresent
+
+  metrics:
+    enabled: true
+    config:
+      receivers:
+        telegraf:
+          agent_config: |
+            [agent]
+              interval = "2s"
+              flush_interval = "3s"
```

Comment:
Maybe `flush_interval` should be set to `interval`, WDYT?

Reply:
yes, and it should be 30 seconds I think :sweat_smile:

Reply:
done :)